static/usage/range/ion-change-event/angular/app_component_html.md

```diff
@@ -0,0 +1,6 @@
+```html
+<div>
+  <ion-range (ionChange)="onIonChange($any($event))"></ion-range>
```

Comment:
I like it! Swapped out for your recommendation. Hopefully we can update to the latest version of Stencil and fix this bug at the source to avoid casting in the near future 👍